src/ui/components/CollectionFilters/CollectionFilters.tsx

```diff
@@ -109,6 +117,17 @@ export const CollectionFilters = React.memo<Props>(
             [handleChangeFilters],
         );
 
+        const handleChangeView = React.useCallback(
+            (value) => {
+                handleChangeFilters({
+                    viewMode: value,
```

Comment:
`viewMode` is not a filter.
This solution leads to refresh of content after changing view mode. It is not correct.